README.md

```diff
@@ -28,13 +28,12 @@ The [PSPDFKit SDK](https://pspdfkit.com/) is a framework that allows you to view
 #### Requirements
 
 - Xcode 10.2.1
-- PSPDFKit 8.4.0 for iOS or later
-- react-native >= 0.59.9
+- PSPDFKit 8.4.2 for iOS or later
+- react-native >= 0.60.3
```

Comment:
So we don't support 0.59.9 at all anymore? We should rather write down all supported versions and mention these steps are for setting it up with 0.60.3

Reply:
I think the README should have the instructions for the latest version of React Native, as most people, when they try out our wrapper, will create a new app, and they'll be using the latest version of React Native.

If someone needs to use 0.59.9, we could always access the old instructions in the git history.